### PR TITLE
Add module for chaos engine

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
The chaos engine module refers the infra needed for the Mattermost Applications
[chaos engine](https://github.com/mattermost/mattermost-app-chaosengine).

#### Ticket Link
Ticket: https://mattermost.atlassian.net/browse/MM-38884

### CODE REVIEW

**Review by stylianosrigas — APPROVED** [review]
LGTM

File: terraform/aws/modules/chaos-engine/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| <a name="input_db_deletion_protection"></a> [db\_deletion\_protection](#input\_db\_deletion\_protection) | n/a | `bool` | `true` | no |
+| <a name="input_db_engine_version"></a> [db\_engine\_version](#input\_db\_engine\_version) | The engine version to use | `string` | `"13.3"` | no |
+| <a name="input_db_identifier"></a> [db\_identifier](#input\_db\_identifier) | The database identifier | `string` | n/a | yes |
+| <a name="input_db_instance_class"></a> [db\_instance\_class](#input\_db\_instance\_class) | The instance type of the RDS instance | `string` | `"t2.micro"` | no |
+| <a name="input_db_maintenance_window"></a> [db\_maintenance\_window](#input\_db\_maintenance\_window) | The window to perform maintenance in. Syntax: 'ddd:hh24:mi-ddd:hh24:mi'. Eg: 'Mon:00:00-Mon:03:00' | `string` | `"Sun:02:00-Sun:03:00"` | no |
+| <a name="input_db_master_az"></a> [db\_master\_az](#input\_db\_master\_az) | The availability zone for Master Database of provisioner | `string` | n/a | yes |
+| <a name="input_db_name"></a> [db\_name](#input\_db\_name) | The DB name to create. If omitted, no database is created initially | `string` | n/a | yes |
```
Comment (angeloskyratzakos) [inline terraform/aws/modules/chaos-engine/README.md:40]: if this omitted it shouldn't be required, right?

File: terraform/aws/modules/chaos-engine/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| <a name="input_db_password"></a> [db\_password](#input\_db\_password) | Password for the master DB user. | `string` | n/a | yes |
+| <a name="input_db_username"></a> [db\_username](#input\_db\_username) | Username for the master DB user | `string` | n/a | yes |
+| <a name="input_environment"></a> [environment](#input\_environment) | The name of the environment will run | `string` | n/a | yes |
+| <a name="input_private_subnets"></a> [private\_subnets](#input\_private\_subnets) | The list of private subnet IDs | `list(string)` | n/a | yes |
+| <a name="input_region"></a> [region](#input\_region) | n/a | `any` | n/a | yes |
```
Comment (angeloskyratzakos) [inline terraform/aws/modules/chaos-engine/README.md:45]: `region` should be string, right?
```suggestion
| <a name="input_region"></a> [region](#input\_region) | n/a | `string` | n/a | yes |
```

File: terraform/aws/modules/chaos-engine/README.md (hunk 1 above)
Comment (spirosoik) [reply terraform/aws/modules/chaos-engine/README.md:40]: why not?

**Review by angeloskyratzakos — APPROVED** [review]
LGTM

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @spirosoik

**Pull request merged**